modules/scenegraph/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@luma.gl/scenegraph",
+  "version": "7.0.0-alpha.15",
+  "description": "Scenegraph classes and glTF support for luma.gl",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/uber/luma.gl"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "webgl",
+    "glsl",
+    "debug",
+    "3d"
+  ],
+  "main": "dist/es5/index.js",
+  "module": "dist/esm/index.js",
+  "esnext": "dist/es6/index.js",
+  "files": [
+    "src",
+    "dist",
+    "README.md"
+  ],
+  "sideEffects": false,
+  "browser": {
+    "fs": false
+  },
+  "scripts": {
+    "clean": "rm -fr dist && mkdir -p dist/es5 dist/esm dist/es6",
+    "build": "npm run clean && npm run build-es6 && npm run build-esm && npm run build-es5",
```

Comment:
These should no longer be necessary

Reply:
> These should no longer be necessary

They are present in all modules, let's update in separate diff given the size of this one.